lua/telescope/make_entry.lua

```diff
@@ -680,6 +680,41 @@ function make_entry.gen_from_highlights()
   end
 end
 
+function make_entry.gen_from_buffer_lines(opts)
+  local displayer = entry_display.create {
+    separator = ' │ ',
+    items = {
+      { width = 5 },
+      { remaining = true },
+    },
+  }
+
+  local make_display = function(entry)
+    return displayer {
+      { entry.lnum, opts.lnum_highlight_group or 'TelescopeResultsSpecialComment' },
+      entry.line
+    }
+  end
+
+  return function(entry)
+    if opts.skip_empty_lines and string.match(entry.line, '^$') then
+      return {
+        valid = false,
+      }
+    end
```

Comment:
You can just do return here. So basically not creating a table and it will be skipped